.github/ISSUE_TEMPLATE/feature_request.yaml

```diff
@@ -0,0 +1,41 @@
+name: Feature request
+description: Suggest an idea for FreeTube which you would like to see in a future release
+title: "[Feature Request]: "
+labels: "enhancement"
+body:
+  - type: markdown
+    attributes:
+      value: |
+        **README: Before You Submit Your Issue**
+        - Issues are not a place to go ask support questions or start discussions. Please ask support questions or start discussions on the [discussions page](https://github.com/FreeTubeApp/FreeTube/discussions).
+  - type: checkboxes
+    attributes:
+      label: Guidelines
+      description: Please ensure you've completed all of the following.
+      options:
+          - label: I have searched the [issue tracker](https://github.com/FreeTubeApp/FreeTube/issues) for a feature request that matches the one i want to file, without success.
```

Comment:
change i to I

Reply:
i think im really blind. Where exactly?

Reply:
![image](https://user-images.githubusercontent.com/72892531/134578382-854d92f0-f6cb-4cfc-bcb0-df06e8b1d3b7.png)
Same for other instance

Reply:
hmm i dont understand please explain why this needs to be a capital I

Reply:
@efb4f5ff-1298-471a-8973-3d47447115dc 
The word "I" is always supposed to be a capital letter

Reply:
oh really didnt know about that. will fix!

Reply:
fixed!